### DISCUSSION TITLE
[IOS] Cannot determine the path to Xcode.app from the sdk root '/Library/Developer/CommandLineTools'.

### DISCUSSION THREAD

**Discussion by bstrother1** [unoplatform/uno#9163] [General]
The Apple MacOS build goes by the Xcode set in Preferences but it seems like the iOS version goes by another variable. How do you set this? It says to set it in the .csproj but what exactly am I looking for? Documentation linked would be appreciated!

**Comment by jeromelaban** [comment]
This message generally means that you need to install the command line tools for Xcode.
